tools/accuracy_checker/accuracy_checker/adapters/pose_estimation_associative_embedding.py

```diff
@@ -0,0 +1,303 @@
+"""
+Copyright (c) 2020 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import numpy as np
+from scipy.optimize import linear_sum_assignment
+
+from ..adapters import Adapter
+from ..config import ConfigValidator, StringField, ConfigError
+from ..representation import PoseEstimationPrediction
+from ..utils import contains_all
```

Comment:
scipy is trouble maker for some VPU platforms, I understand that it gives you performance boost, but its usage should not break the tool usage for platforms where scipy is not available (it is the reason why we have it as soft requirement, the same  thing, I can say about skimage).

```suggestion

from ..adapters import Adapter
from ..config import ConfigValidator, StringField, ConfigError
from ..representation import PoseEstimationPrediction
from ..utils import contains_all, UnsupportedPackage

try:
    from scipy.optimize import linear_sum_assignment
 except ImportError as error:
     linear_sum_assignment = UnsupportedPackage('scipy.optimize', error.msg)
```